extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -53,6 +53,11 @@ export class PlatformStore {
 
     /** @private {!Array<string>} */
     this.failedPlatforms_ = [];
+
+    /** @private @const {!Object<string, number>} */
+    this.scoreConfig_ = Object.assign({
+      'supportsViewer': 9,
```

Comment:
Now that it's public, let's make this a round value - `10`. It's documented as `10` anyway :)